src/ln/channelmanager.rs

```diff
@@ -3126,7 +3128,9 @@ mod tests {
 				}
 			}
 		}
-		assert_eq!(res.len(), 1);
+		if revoked_tx.is_none() {
```

Comment:
Why not just check against revoked is_none() and PenaltyType is HTLC so you check the call is doing *something* (and doesn't change existing logic).

Reply:
Modified, but you mean against revoked_is_some() right?